Cython/Compiler/Nodes.py

```diff
@@ -8200,7 +8230,7 @@ def analyse_expressions(self, env):
         return self
 
     def generate_execution_code(self, code):
-        pass
+        cimport_numpy_check(self, code)
 
 
 class FromCImportStatNode(StatNode):
```

Comment:
I think we're missing a test for `from numpy cimport …`.

Reply:
Yes - those tests were actually failing so good catch. The `FromCImportStatNode` nodes were being dropped in `MarkParallelAssignments` - (I think unintentionally)